src/index.css

```diff
@@ -1,9 +1,15 @@
 :root {
   --primary-color: #2232cf;
   --text-color: black;
+  --white-color: #fff;
+  --icons-color: #5f5e5e;
```

Comment:
try to use only one convention for colors

Reply:
Okay